angular.json

```diff
@@ -5332,7 +4989,8 @@
                     }
                 }
             },
-            "tags": ["fdp"]
+            "tags": ["fdp"],
+            "implicitDependencies": ["core"]
```

Comment:
do we really need to keep implicitDependencies on all other packages except for those 3 that are failing due to unknown reason?

Reply:
After fixing those three packages build started failing on `platform-form` and so on, so i think order of packaging is broken in nx.